File_handling/fileOperations.test.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs')
+
+const {persons, saveDataToFile} = require('./fileOperations.js')
+
+test('Reading data from file', () => {
+    const readMe = 'persons.json'
```

Comment:
Why this var is called readme?

Reply:
To make reference to the file that will be read.

Reply:
Ah I see, but the names is not clear, maybe you should call it personFilename or something

Reply:
Great tip, thank you.